.gitignore

```diff
@@ -143,6 +143,7 @@ __pycache__
 
 # Compiled JavaScript
 pipeline/build/
+backend/build/
```

Comment:
Can these two be replaced by `**/build/`?

Reply:
I did this in a different PR, so yes, this is `**/build` now.